runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayLayer.java

```diff
@@ -27,17 +27,22 @@
 public enum OverlayLayer
 {
 	/**
-	 * Render overlay above all game elements
+	 * Render right above surface with actors and objects
 	 */
-	ALWAYS_ON_TOP,
+	ABOVE_SURFACE,
```

Comment:
The consensus seems to be to rename this to ABOVE_SCENE

Reply:
Done and updated